packages/core/package.json

```diff
@@ -29,9 +29,9 @@
     },
     "./storybookComponents": "./dist/storybook/index.js",
     "./next": {
-      "types": "./dist/esm/next/next.d.ts",
-      "import": "./dist/esm/src/next/next.js",
-      "default": "./dist/esm/src/next/next.js"
+      "types": "./dist/esm/components/next/next.d.ts",
```

Comment:
There were two files which one of them was redundant, so now I pointed it to the other file (makes more sense)